src/hidpp20.c

```diff
@@ -1776,6 +1777,29 @@ hidpp20_buttons_from_cpu(struct hidpp20_profile *profile,
 	}
 }
 
+static void
+hidpp20_onboard_profile_read_led(struct hidpp20_led *led,
+			struct hidpp20_internal_led internal_led)
+{
+	uint16_t rate = 0;
+	uint8_t brightness = 0;
+	switch (internal_led.mode) {
```

Comment:
empty line here please. and I think the second argument above is not aligned, but that could be github's display